.changeset/smart-buttons-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-backend': minor
```

Comment:
This can be `patch` for now!  While we're on `0.x.y` releases, we're using `minor` for breaking changes.